### DISCUSSION TITLE
Feature request: Disable Tabs in the File Explorer

### DISCUSSION THREAD

**Discussion by jed84** [valinet/ExplorerPatcher#2197] [Ideas]
**Request:**
This is a request to allow the users to disable the tabs in the File Explorer.

**Reason:**
The current implementation of tabs in the File Explorer breaks the solutions that are alredy offered by the 3rt parties (for example QTTabBar).
In order to make the other solutions work, we need to be able to disable the tabs in the File Explorer.

**Notes:**
Previously ViveTool 0.3.3 allowed to disable the Tabs in the File Explorer by executing the following commands:
```
vivetool /disable /id:36354489
vivetool /disable /id:37634385
vivetool /disable /id:39145991
```
This no longer works on Windows 11 22H2 (22621.2361) and executing the above codes will result in the File Explorer crash.

**Comment by Amrsatrio** [comment]
Do you still need the Windows 11 command bar? If not, you can just enable the Windows 10 or Windows 7 command bar.

Also, speaking about versions, because 22H2 updates monthly with new features (that sometimes breaks EP too), it's better if you specify the last part of the version such as 2361, so like 22621.2361.

**Reply by Erz888** [reply]
hello Amrsatrio, I have same problem to disable multi-tabs in version Win11 23H2. Do you know what ID disable to close multi-tabs? I wanna use QTTabBar too in 23H2. thank you very much.

**Comment by jed84** [comment]
I have added the missing details. It's 22H2 22621.2361.

Thank you for the tip 💚 - as for now I enabled Win 10 ribbon.

Hopefully a better solutione comes up soon:
I wish to disable the tabs in the File Explorer (as I am no longer able to user QTTabBar), but at the same time I would like to keep the modern Win 11 command bar.

**Reply by pyrates999** [reply]
That is a preview update: https://support.microsoft.com/en-us/topic/september-26-2023-kb5030310-os-build-22621-2361-preview-363ac1ae-6ea8-41b3-b3cc-22a2a5682faf

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

Preview updates are not supported by EP.

**Reply by Erz888** [reply]
hello, I have same problem to disable multi-tabs in version Win11 23H2. Do you know what ID disable to close multi-tabs? I wanna use QTTabBar too in 23H2. thank you very much.

**Reply by pyrates999** [reply]
The id's to disable tabs have been listed here already.

**Reply by Erz888** [reply]
Thanks for kindly response. I tried disable ID 36354489, 37634385 and 39145991, it's result in the File Explorer crashed. Is there any ID can disable file explore multi-tabs in 23H2 Windows 11? Also, I returned to 22H2, It's doesn't work too, but I can disable it  in 22H2 last year.

**Reply by pyrates999** [reply]
yeah, with the latest update, I don't think there's a way to disable tabs without explorer.exe crashing, sorry.

**Reply by Erz888** [reply]
okay, never mind. thank you. If you figure it out someday, please let me know. thank you again for answered me.

**Comment by Amrsatrio** [comment]
I disabled 37634385 (TIFE "Tabs in File Explorer") and Explorer crashed on 23545.1000 (yes I know it's a non release build but it's easier to mess with) when you try to open a folder. I guess let's keep this suggestion for another time since Microsoft keeps changing stuff. There are way more important issues to look at right now given how many people care about the development of this project.

**Comment by wany-oh** [comment]
In 22H2, disabling `40729001` in addition to `37634385` will work.

**Reply by DyonR** [reply]
Also confirmed to be working for 22631.2506; `vivetool /disable /id:40729001,37634385`

**Reply by pyrates999** [reply]
Please list your windows version that you ran this on so we know which minimum version it works on.

**Reply by Amrsatrio** [reply]
@DyonR Can you send a screenshot? My 22621.2506 machine shows the Windows 10 ribbon but with always light mode title bar.

**Comment by Amrsatrio** [comment]
I've done some research here, and found out that the feature flag 37634385 (TIFE "Tabs in File Explorer") has been removed in 23575 (Dev), therefore I don't know if we can go ahead with your suggestion in the long term.
